mmcv/parallel/distributed.py

```diff
@@ -18,7 +18,7 @@ class MMDistributedDataParallel(DistributedDataParallel):
     - It implement two APIs ``train_step()`` and ``val_step()``.
     """
 
-    def to_kwargs(self, inputs, kwargs, device_id):
+    def to_kwargs(self, inputs, kwargs, device_id: int):
```

Comment:
According to `mmcv.parallel.scatter_gather.scatter`, inputs should be `Union[DataContainer, Tensor, dict, tuple, list]`

Reply:
Type hints for return value